File: app/android/app/src/main/java/com/example/arken/model/Article.kt
package com.example.arken.model

import java.io.Serializable
import java.util.Date

class Article (
    val _id: String? = null,
    val rateAverage: Double? = null,
    val numberOfRates: Int? = null,
    val userId: String? = null,
    val date: Date? = null,
    val __v: Int? = null,
    val username: String? = null,
    val usersurname: String? = null,
    val yourRate: Int? = null,
    val title: String? = null,
    val text: String? = null,
    var comments: MutableList<Comment>? = null,
    var imageId:Int = 0
    ) : Serializable

File: app/backend/models/article.js
const {mongoose} =  require('../db');  // The mongodb connector library

let Article = mongoose.model('Article', {

  userId: {
    type: String,
    require: true
  },

  text: {
    type: String,
    require: true
  },

  title:{
      type: String,
      require: true
  },

  date:{
    type: Date,
    require: true
  },

  rateAverage:{
    type: Number,
    default: 0
  },

  numberOfRates:{
    type: Number,
    default: 0
  }
});

module.exports = {
  Article: Article
}